functions/tables.py

```
@@ -0,0 +1,132 @@
+        rows = conn.execute(stmt).mappings().all()
+    return [{key: _encode_value(value) for key, value in row.items()} for row in rows]
+
+
+def create_table_row(table_name: str, values: Dict[str, Any]) -> Dict[str, Any]:
+    table = _get_table(table_name)
+    prepared: Dict[str, Any] = {}
+    for column in table.c:
+        if column.name in values and not column.primary_key:
+            prepared[column.name] = _decode_value(column, values[column.name])
+    if not prepared:
+        raise ValueError("Inga giltiga kolumner angavs")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)